backend/src/main/java/com/woowacourse/gongseek/auth/application/AuthService.java

```diff
@@ -0,0 +1,44 @@
+package com.woowacourse.gongseek.auth.application;
+
+import com.woowacourse.gongseek.auth.presentation.dto.OAuthCodeRequest;
+import com.woowacourse.gongseek.auth.presentation.dto.OAuthLoginUrlResponse;
+import com.woowacourse.gongseek.auth.presentation.dto.TokenResponse;
+import com.woowacourse.gongseek.member.domain.Member;
+import com.woowacourse.gongseek.member.domain.repository.MemberRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Transactional
+@Service
+public class AuthService {
+
+    private final GithubOAuthClient githubOAuthClient;
+    private final MemberRepository memberRepository;
+    private final JwtTokenProvider jwtTokenProvider;
+
+    public OAuthLoginUrlResponse getLoginUrl() {
+        return new OAuthLoginUrlResponse(githubOAuthClient.getRedirectUrl());
+    }
+
+    public TokenResponse generateAccessToken(OAuthCodeRequest OAuthCodeRequest) {
+        Member member = githubOAuthClient.getMemberProfile(OAuthCodeRequest.getCode()).toMember();
```

Comment:
Article Request 처럼 toEntity 어떤가요?
MemberProfile을 받아서 Member로 변환한다는게 뭔가 어색하네요.
네이밍이 고민을 많이 하게 하니까 Dto -> Entity 는 다 toEntity로 통일하는거 어떤가요?
```suggestion
        Member member = githubOAuthClient.getMemberProfile(OAuthCodeRequest.getCode()).toEntity();
```